contents/tutorials/cohorts.md

```diff
@@ -1,121 +0,0 @@
-
-Each person uses your product slightly differently from the next, and some groups might use your app in a completely different way from others.
-
-This happens for a variety of reasons, including software versions, user profiles, accessibility needs, culture, understanding of the product, and individual goals, among many other variables.
-
-Cohort analysis is an established technique for performing analytics, where users are split into different cohorts according to certain characteristics. You can then evaluate metrics for each specific cohort, rather than performing your analytics on the entire collection of users.
-
-Here are some examples:
-
-**Segmenting by engagement level**
-
-One useful way of grouping users is by their engagement with your product. How do power users use your product differently? 
-
-**Segmenting by profile data**
-
-When users sign up, you'll often gather information about them such as their role. For example, understanding how people in different roles use your product can help you work out how to solve for their unique needs.
-
-**Segmenting by previous behavior**
-
-Users who performed a certain defining action might use your product differently from those who haven't. At PostHog, for example, we have a dedicated cohort for people who have accessed our Docs, as we see users who have used them are more likely to be successful.
-
-**Segmenting by marketing source**
-
-PostHog grabs [UTM tags](https://en.wikipedia.org/wiki/UTM_parameters) from URLs by default, as well as capturing information such as 'Referrer URL' and 'Initial Referrer Domain' as properties on Events. This allows you to segment users based on where they came from, so you can determine if users from a certain marketing campaign or channel is working better than another.
-
-**Segmenting by browser or device**
-
-This allows you to determine if your UX is problematic on certain browsers. This can be indicated by a lower retention, worse conversion in your Funnels, or certain Events not being captured. A common issue is desktop web products not rendering correctly for users on mobile devices.
-
-### Defining a group of users: Setting up a Cohort in PostHog
-
-User group segmentation for cohort analysis is built into PostHog via our Cohorts feature. Cohorts let you group users by properties in common, as well as by actions over a time period.
-
-For example, you could define the following cohorts:
-
-- Users who did action 'Sign Up' in the last month
-- Users with property `email` containing `@posthog.com` _or_ property `company` equal to `PostHog`
-- Users who have property `role` equal to `Software Engineer` _or_ triggered action `Used API` in the last 7 days
-
-These cohorts can then be used in 'Insights' to compare metrics across groups. 
-
-**Creating a cohort**
-
-To create a cohort, click on 'Cohorts' on the sidebar and then click '+ New Cohort'. 
-
-![Cohorts Page](../images/tutorials/cohorts/cohorts-page.png)
```

Comment:
Should we delete these images too?

Reply:
Probably safer not to, just in case they're used somewhere else as well.